internal/spentenergy/spentenergy.go

```diff
@@ -2,6 +2,8 @@ package spentenergy
 
 import (
 	"time"
+
+	usererrors "github.com/Yandex-Practicum/tracker/internal/usererrors"
```

Comment:
нет смысла называть пакет так как он уже так и называется
```suggestion
	"github.com/Yandex-Practicum/tracker/internal/usererrors"
```